flytekit/bin/entrypoint.py

```diff
@@ -249,6 +273,14 @@ def get_traceback_str(e: Exception) -> str:
     return format_str.format(traceback=tb_str, message=f"{type(value).__name__}: {value}")
 
 
+def get_timestamp(e: Optional[Exception] = None) -> Timestamp:
```

Comment:
```suggestion
def get_container_error_timestamp(e: Optional[Exception] = None) -> Timestamp:
    """Get timestamp for ContainerError.
    
    If a flyte exception is passed, use its timestamp, otherwise, use the current time.
    
    Parameters
    ----------
    e : Exception, optional
        Exception that has occured.
    
    Returns
    -------
    Timestamp
        Timestamp to be reported in ContainerError
```

Reply:
Sounds good